contracts/scripts/native_solc_compile_all

```diff
@@ -9,6 +9,7 @@ solc-select install 0.6.6
 solc-select install 0.7.6
 solc-select install 0.8.6
 solc-select install 0.8.15
+yarn install
```

Comment:
```suggestion

```

We shouldn't need this, the native compiles use `solc-select` which is a uses the native solc compiler

Reply:
I guess the README needs to be updated to reflect that we use `pnpm` now, but the user is expected to run `pnpm install` as part of repo setup, we shouldnt need to hook this into the generate command

Reply:
Related: https://github.com/smartcontractkit/chainlink/pull/7775

Reply:
Excellent!  Glad I asked